frequency-plans.yml

```
@@ -161,6 +161,27 @@
   country-codes: [jp, my, sg]
   file: lbt_80_over_128.yml
 
+- id: AS_920_923_JP_TTN_1
+  name: Japan 920-923 MHz with LBT(CH31-CH38)
+  description: Frequency plan for Japan, using continuous frequencies up to 923.4MHz with LBT.
+  base-frequency: 915
+  country-codes: [jp]
+  file: AS_920_923_JP_TTN_1.yml
+
+- id: AS_920_923_JP_TTN_2
+  name: Japan 920-923 MHz with LBT(CH24-27,CH35-CH38)
+  description: Frequency plan for Japan, using top and bottom frequencies ≤ 923.4 MHz with LBT.
+  base-frequency: 915
+  country-codes: [jp]
+  file: AS_920_923_JP_TTN_2.yml
+
+- id: AS_920_923_JP_TTN_3
+  name: Japan 920-923 MHz with LBT(CH24-31)
```

```suggestion
  name: Japan 920-923 MHz with LBT(channels 24-31)
```

Also is it inclusive or not, because if this one includes 31 and the first one too, there's a duplicate right?

---

I intentionally included CH31 in AS_920_923_JP_TTN_3, because frequencies <= 923.4MHz in Japan are 15 channels,
The Kerlink's 16 channels gateway have 2 8-channel radio, so I have to design 2 continuous 8 channels to cover all of 15 channels.

FYI) In Japan, under 923.5MHz(CH24-38) and over 923.5MHz(CH39-) have different spec in ARIB STD-T108.
   Please see attached Table 3-18 and Table 3-8.